notebooks/StatiGraphics.jl

```diff
@@ -0,0 +1,164 @@
+### A Pluto.jl notebook ###
+# v0.12.20
+
+using Markdown
+using InteractiveUtils
+
+# ╔═╡ 01c88650-1fa3-11eb-1156-bb632230292b
+using Random; Random.seed!(2019); # make sure this tutorial is reproducible
+
+# ╔═╡ 221618e0-6e73-11eb-047e-3b329eefa7d3
+begin
+	using GeoStats
+	using StratiGraphics
+	using SpectralGaussianSimulation
+	
+	solver₁ = FFTGS(:land => (variogram=GaussianVariogram(range=100.,sill=3e-2),))
+	solver₂ = FFTGS(:land => (variogram=GaussianVariogram(range=100.,sill=3e-2),))
+	
+	procs = [GeoStatsProcess(solver) for solver in [solver₁, solver₂]];
+end
+
+# ╔═╡ cc7eef10-6e72-11eb-2cd4-abf33bd7f1ad
+md"""
+# Installation & Setup
+
+Before we proceed, please install the following packages:
+"""
+
+# ╔═╡ 6c9b41d8-043f-4626-8798-4a3ae9da922b
+md"""
+# Creating surfaces
+
+First, we demonstrate how Markov-Poisson sampling can be used to generate random surfaces of a 3D stratigraphic model.
+
+We start by defining a set of 2D geostatistical processes:
```

Comment:
Is there something missing here, why the colon following by another paragraph?

Reply:
I think it's correct. The original notebook also has the same structure.